test/handle-multiple-registrations.html

```diff
@@ -19,11 +19,15 @@
   <script type="module">
     import '../test-fixture.js';
     import './test-fixture-symlink.js';
+    window.IMPORTS_DID_COMPLETE = true;
+  </script>
+  <!-- Load in a new script tag, in case the module graph in the above tag
+  fails. -->
```

Comment:
If these aren't separated, then the `test` function below won't run if an uncaught exception is thrown somewhere when loading the modules before it. Although, this might not matter, assuming mocha is paying attention to uncaught exceptions. Also, when `test-fixture-symlink.js` tries to register a second time, if it doesn't catch the error, then the tree of modules imported in the first script will bail out at that point even though the first registration of `test-fixture` was successful, so the `IMPORTS_DID_COMPLETE` thing makes sure that it actually caught the error. (i.e. `test-fixture-symlink.js` could throw and `customElements.get('test-fixture')` will still return the right value in the test.)

Reply:
Oh, are you saying that I should just put them in one and let mocha notice the uncaught error? Also, what's going to cause this to flake?

Reply:
Yeah, that works and is simpler. I'll push an update.